### ISSUE TITLE
Invalid parameter neurons for estimator KerasClassifier.

### ISSUE THREAD

**Issue by AntonioConstantino** [adriangb/scikeras#318]
Check the list of available parameters with `estimator.get_params().keys()`

def build_model(optimizer='adam', neurons=60):
    model=Sequential()
    model.add(Embedding(NUM_PALABRAS, neurons, input_length=longitud_maxima))
    model.add(Dropout(0.2))
    model.add(Flatten())
    model.add(Dense(12, activation='relu'))
    model.add(Dropout(0.3))
    model.add(Dense(len(generos), activation='softmax'))
    model.compile(loss='categorical_crossentropy', optimizer=optimizer, metrics=['accuracy'])
    return model

neurons = [30, 60, 90]
batch_size = [10]
optimizer = ['adam', 'rmsprop']

param_grids = dict(neurons=neurons, batch_size=batch_size, optimizer=optimizer)
model = KerasClassifier(build_fn=build_model, verbose=2)

grid = GridSearchCV(estimator=model, param_grid=param_grids, n_jobs=-1)
result = grid.fit(X_train, y_train,)

print("Best: {} using {}".format(result.best_score_, result.best_params_))
